src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -152,8 +152,20 @@ export default class SourceFileLinter {
 					return parentClassType.symbol?.declarations?.flatMap((declaration) => {
```

Comment:
off topic: what is the `flatMap` needed for? Why was `some` not sufficient?

Reply:
Can't recall the exact reasoning, but after look at it, definitelly `some` would work perfectly fine here

Reply:
I kept the `flatMap` during my refactoring as I wasn't sure whether it might be helpful in case we want to collect information about the inheritance. But right now I don't think that it is necessary, so using `some` would make sense to me.

Reply:
https://github.com/SAP/ui5-linter/pull/366/commits/37f9ff27e90dbea8144e386442f0239493d926b1